core/src/main/java/org/apache/iceberg/UpdateSnapshotReferencesOperation.java

```diff
@@ -106,40 +106,42 @@ public UpdateSnapshotReferencesOperation replaceBranch(String name, long snapsho
     return this;
   }
 
-  public UpdateSnapshotReferencesOperation replaceBranch(String name, String source) {
-    return replaceBranch(name, source, false);
+  public UpdateSnapshotReferencesOperation replaceBranch(
+      String branchToUpdate, String destination) {
```

Comment:
do we want to align this with the parameter naming in `ManageSnapshots replaceBranch(String from, String to)`?

Reply:
Yes, done.